verl/trainer/ppo/ray_trainer.py

```diff
@@ -969,9 +972,9 @@ def fit(self):
                             batch = batch.union(reward_tensor)
 
                         if self.config.reward_model.launch_reward_fn_async:
-                            future_reward = compute_reward_async.remote(batch, self.config, self.tokenizer)
+                            future_reward = compute_reward_async.remote(batch, self.config, self.tokenizer, tracker=self.tracker, step=self.global_steps)
                         else:
-                            reward_tensor, reward_extra_infos_dict = compute_reward(batch, self.reward_fn)
+                            reward_tensor, reward_extra_infos_dict = compute_reward(batch, self.reward_fn, tracker=self.tracker, step=self.global_steps)
```

Comment:
Return all the necessary information to calculate positive/negative seqlen, and then add to logger here.